src/encoded/static/components/item-pages/components/WorkflowDetailPane/FileDetailBody.js

```diff
@@ -22,108 +23,41 @@ export class FileDetailBody extends React.Component {
         'minHeight' : PropTypes.number,
         'keyTitleDescriptionMap' : PropTypes.object,
         'windowWidth' : PropTypes.number.isRequired
-    }
+    };
 
     static defaultProps = {
         'canDownloadStatuses' : fileUtil.FileDownloadButtonAuto.defaultProps.canDownloadStatuses
-    }
-
-    constructor(props){
-        super(props);
-        this.componentDidMount = this.componentDidMount.bind(this);
-        this.maybeLoadFile = this.maybeLoadFile.bind(this);
-        this.state = {
-            file : this.props.file
-        };
-    }
-
-    componentDidMount(){
-        this.maybeLoadFile();
-    }
-
-    componentWillReceiveProps(nextProps){
-        if (nextProps.file !== this.props.file) {
-            this.setState({ file : nextProps.file }, this.maybeLoadFile.bind(this, nextProps.file));
-        }
-    }
-
-    maybeLoadFile(file = this.state.file){
```

Comment:
We remove this function as well as all logic relating to it in order to simplify.
We instead add a little bit more info to the File representation in the Workflow/Provenance graph tracing, in place of trying to AJAX file each time.